### PR TITLE
Update Conv-Add-Relu Fusion Transformation

### PR DESCRIPTION
**Pull request by jchen351**
…n name to make it more intuitive.

### Description
Update Conv-Add-Relu Fusion Transformation to handle additional case where NhwcFusedConv is present.

### Motivation and Context
Handle additional case where NhwcFusedConv is present.

### CODE REVIEW

File: onnxruntime/core/optimizer/conv_add_act_fusion.cc [hunk 1]
```diff
@@ -122,7 +124,7 @@
     return is_supported_cpu_ep_activation(activation_node);
   }
 
-  const Node* SelectProducerConv(const Node& node) const {
+  const Node* SelectProducerConv(const Node& node, bool isNhwc) const {
```
Comment (chenfucn) [inline onnxruntime/core/optimizer/conv_add_act_fusion.cc:127]: >isNhwc [](http://example.com/codeflow?start=56&length=6)

is this necessary?

Comment (jchen351) [reply onnxruntime/core/optimizer/conv_add_act_fusion.cc]: Nice catch, I am using runtimeState.selected_nodes.Target().OpType(); for OpType() function from FuseConvAddActivationAction, which has eliminated the need for additional boolean flag as well.

**Review by chenfucn — APPROVED** [review]

File: onnxruntime/core/optimizer/conv_add_act_fusion.cc [hunk 2]
```diff
@@ -176,8 +175,8 @@
       // Check if this is a single use convolution that hasn't already
       // been fused with another Add/Sum node. The Add/Sum can also only be
       // fused if the convolution isn't itself fused with an activation.
-      if ((inputs_node[n]->OpType() == "Conv") && (pre_input_defs_count < 4) && (producer_input_args_count.size() < 4) &&
-          (graph_utils::GetNodeAttribute(*inputs_node[n], "activation") == nullptr) && (inputs_node[n]->GetOutputEdgesCount() == 1)) {
+      if (
+          (inputs_node[n]->OpType() == "Conv" || inputs_node[n]->OpType() == "NhwcFusedConv") && (pre_input_defs_count < 4) && (producer_input_args_count.size() < 4) && (graph_utils::GetNodeAttribute(*inputs_node[n], "activation") == nullptr) && (inputs_node[n]->GetOutputEdgesCount() == 1)) {
```
Comment (yufenglee) [inline onnxruntime/core/optimizer/conv_add_act_fusion.cc:179]: nit: ah, this line is too long. Does it pass the format check?

Comment (jchen351) [reply onnxruntime/core/optimizer/conv_add_act_fusion.cc:179]: Yes, actually I used the formatter provided by clang

**Pull request merged**